packages/react-composites/src/composites/CallComposite/components/buttons/People.tsx

```diff
@@ -15,6 +17,14 @@ const icon = (): JSX.Element => <CallCompositeIcon iconName={'ControlButtonParti
 /** @beta */
 export const People = (props: ControlBarButtonProps): JSX.Element => {
   const { strings, onRenderOnIcon, onRenderOffIcon, onClick } = props;
+
+  /* @conditional-compile-remove(rooms) */
```

Comment:
This line shouldn't be conditionally complied?

Reply:
We are spreading props directly to ControlButton (on line 45) in stable. So isn't this line is not needed in stable?

Reply:
I think we only need to conditionally compile the permissions statement `disabled = disabled || !_usePermissions().participantList`

Reply:
But the line: 
```let disabled = props.disabled;```
is not needed in stable.

Reply:
Got it, so it's all beta feature even disable prop